File: RESTful API Gin Gonic/src/entities/employee.go
package entities

import "time"

type Employee struct {
	EmployeeId  uint    `json:"employee_id" gorm:"type:bigint;primary_key,AUTO_INCREMENT:true;index"`
	FirstName   string  `json:"first_name" gorm:"type:varchar(255)"`
	LastName    string  `json:"last_name" gorm:"type:varchar(255)"`
	Email       string  `json:"email" gorm:"type:varchar(255)"`
	Gender      string  `json:"gender" gorm:"type:varchar(255)"`
	DateOfBirth time.Time `json:"date_of_birth"`
	Location    string  `json:"location" gorm:"type:varchar(255)"`
	Department  string  `json:"department" gorm:"type:varchar(255)"`
	JobTitle    string  `json:"job_title" gorm:"type:varchar(255)"`
	Salary      int     `json:"salary" gorm:"type:integer"`
	DateOfHire  time.Time  `json:"date_of_hire"`
	Picture     string  `json:"picture" gorm:"type:varchar(255)"`
}

func (Employee) TableName() string {
	return "employee"
}

File: RESTful API Express Js/src/entities/employee.ts
import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";

@Entity({ name: "employee" })
class Employee {
  @PrimaryGeneratedColumn({ name: "employee_id" })
  employee_id!: number;

  @Column({ name: "first_name" })
  first_name!: string;

  @Column({ name: "last_name" })
  last_name!: string;

  @Column()
  email!: string;

  @Column()
  gender!: string;

  @Column({ name: "date_of_birth" })
  date_of_birth!: Date;

  @Column()
  location!: string;

  @Column()
  department!: string;

  @Column({ name: "job_title" })
  job_title!: string;

  @Column()
  salary!: number;

  @Column({ name: "date_of_hire" })
  date_of_hire!: Date;

  @Column()
  picture!: string;
}

export default Employee;
